os/test/src-jvm/SubprocessTests.scala

```diff
@@ -104,17 +104,41 @@ object SubprocessTests extends TestSuite {
 
     test("envArgs") {
       if (Unix()) {
-        val res0 = proc("bash", "-c", "echo \"Hello$ENV_ARG\"").call(env = Map("ENV_ARG" -> "12"))
-        assert(res0.out.lines() == Seq("Hello12"))
+        locally {
+          val res0 = proc("bash", "-c", "echo \"Hello$ENV_ARG\"").call(env = Map("ENV_ARG" -> "12"))
+          assert(res0.out.lines() == Seq("Hello12"))
+        }
+
+        locally {
+          val res1 = proc("bash", "-c", "echo \"Hello$ENV_ARG\"").call(env = Map("ENV_ARG" -> "12"))
+          assert(res1.out.lines() == Seq("Hello12"))
+        }
+
+        locally {
+          val res2 = proc("bash", "-c", "echo 'Hello$ENV_ARG'").call(env = Map("ENV_ARG" -> "12"))
+          assert(res2.out.lines() == Seq("Hello$ENV_ARG"))
+        }
 
-        val res1 = proc("bash", "-c", "echo \"Hello$ENV_ARG\"").call(env = Map("ENV_ARG" -> "12"))
-        assert(res1.out.lines() == Seq("Hello12"))
+        locally {
+          val res3 = proc("bash", "-c", "echo 'Hello'$ENV_ARG").call(env = Map("ENV_ARG" -> "123"))
+          assert(res3.out.lines() == Seq("Hello123"))
+        }
 
-        val res2 = proc("bash", "-c", "echo 'Hello$ENV_ARG'").call(env = Map("ENV_ARG" -> "12"))
-        assert(res2.out.lines() == Seq("Hello$ENV_ARG"))
+        locally {
+          val res4 = proc("bash", "-c", "echo \"$TEST_SUBPROCESS_ENV\"").call(
```

Comment:
Can you add a comment, that we expect this env var to be set from the build tool / test runner?

Reply:
Or even assert that it is set as expected.